tests/prefetch_related/models.py

```diff
@@ -65,7 +65,12 @@ class BookWithYear(Book):
 
 
 class Bio(models.Model):
-    author = models.OneToOneField(Author, models.CASCADE)
+    author = models.OneToOneField(
+        Author,
+        models.CASCADE,
+        primary_key=True,
+        to_field='name',
+    )
```

Comment:
Thanks. I checked and so far there was only one usage of this model directly and indirectly (prefetch).